agent/xds/xds_protocol_helpers_test.go

```diff
@@ -647,3 +660,23 @@ func runStep(t *testing.T, name string, fn func(t *testing.T)) {
 		t.FailNow()
 	}
 }
+
+func requireProtocolVersionGauge(
```

Comment:
Do you expect that we will call this from other places? Right now it's only called from one place so it might be good to keep it next to the one call site.

Reply:
It's called in two places: `server_test.go` and `delta_test.go` for v2 and v3 respectively.